tests/detect-flow-pkts/test.rules

```diff
@@ -1 +1 @@
-alert ip any any -> any any (msg:"Flow has 10 packets"; flow.pkts_toclient:10; flow.pkts_toserver:10; flow.bytes_toserver:<3000; flow.bytes_toclient:>8000; sid:1;)
+alert ip any any -> any any (msg:"Flow has 10 packets"; flow.pkts:toclient,=10; flow.pkts:toserver,=10; flow.bytes:toserver,<3000; flow.bytes:toclient,>8000; sid:1;)
```

Comment:
Do you need the `=` in `=10` ?

Reply:
As far as I understood. This mode will work with or w/o the `=`. I'll remove the `=` for simplicity. Thank you. :)